app/waf/src/models/pool.model.ts

```diff
@@ -1,24 +1,19 @@
 import {Member} from './member.model';
-import {Entity, model, property, hasMany} from '@loopback/repository';
+import {model, property, hasMany} from '@loopback/repository';
+import {CommonEntity} from '.';
 
 @model()
-export class Pool extends Entity {
-  @property({
-    type: 'string',
-    id: true,
-  })
-  id: string;
-
-  @property({
-    type: 'string',
-    required: false,
-  })
-  name?: string;
-
+export class Pool extends CommonEntity {
```

Comment:
AS3 pool has 11 properties. 3 of them (class, label, remark) will be handled by CommonEntity class. 2 of them (members and monitors) will be implement by many-to-many relation. Other 6 properties can be implement in this class.

Reply:
I will add them now.